packages/@expo/env/src/env.ts

```diff
@@ -136,7 +160,11 @@ export function createControlledEnvironment() {
       }
     }
 
-    process.env = { ...process.env, ...envInfo.env };
+    for (const key of Object.keys(envInfo.env)) {
+      // Avoid creating a new object, mutate it instead as this causes problems in Bun
```

Comment:
😵‍💫

Reply:
I mean, do you want me to add a link to my tweet as comment? https://twitter.com/cedric_dev/status/1724544717147296070 🙈